File: studies/movies/ggplot2/bar_budget_avg_colored.R
library(tidyverse)

df <- read_csv("data/movies.csv")
ggplot(df) +
  geom_bar(aes(
    x = `Major Genre`, y = `Production Budget`,
    fill = `MPAA Rating`
  ),
  position = "dodge", stat = "summary", fun = "mean"
  )

File: galleries/movies/seaborn/bar_budget_avg_colored.py
import pandas as pd
import seaborn as sns

df = pd.read_csv("data/movies.csv")
ax = sns.barplot(
    df,
    x="Major Genre",
    y="Production Budget",
    hue="MPAA Rating",
    estimator="mean",
    errorbar=None,
)
